README.md

```diff
@@ -1,190 +1,97 @@
-# Header [![Build Status](https://travis-ci.org/Financial-Times/o-header.svg)](https://travis-ci.org/Financial-Times/o-header)
```

Comment:
Are you not using the status API token from Circle CI for the shield or is that unneeded?